lib/jss.map

```diff
@@ -394,3 +394,9 @@ Java_org_mozilla_jss_nss_Buffer_Free;
     local:
         *;
 };
+JSS_4.6.2 {
```

Comment:
I've done a terrible job maintaining this map. @edewata, @jmagne, or @ladycfu -- do you know if it is possible to retroactively update the version numbers that functions were added in? :-)